cognito-post-authentication/src/main/java/no/unit/nva/cognito/util/EventUtils.java

```diff
@@ -0,0 +1,20 @@
+package no.unit.nva.cognito.util;
+
+import java.util.Map;
+
+public final class EventUtils {
```

Comment:
This class could be replaced by an `Event` class with the appropriate Jackson annotations